pkg/util/migrations/migrations.go

```diff
@@ -23,6 +24,29 @@ func ListUnfinishedMigrations(informer cache.SharedIndexInformer) []*v1.VirtualM
 	return migrations
 }
 
+func ListWorkloadUpdateMigrations(informer cache.SharedIndexInformer, vmiName, ns string) []v1.VirtualMachineInstanceMigration {
+	objs := informer.GetStore().List()
+	migrations := []v1.VirtualMachineInstanceMigration{}
+	for _, obj := range objs {
+		migration := obj.(*v1.VirtualMachineInstanceMigration)
+		if migration.IsFinal() {
+			continue
+		}
+		if migration.Namespace != ns {
```

Comment:
can't/shouldn't this ionformer have a "namespace" index?

Reply:
Could also add your own "vmi" index that indexes on namespace/name

Reply:
@mhenriks I'm not sure I understand the feedback, could you please expand?

Reply:
If you want to go further with this approach you can even add a new indexer that would get the aborted migrations and use it here :smiley:

Reply:
do you mind if I address this in a followup PR? I'd like to get this merge in order to continue with volume migration

Reply:
Not at all, as I said it's not a blocker from my side :)